### PR TITLE
Update introduction.tex

### PR DESCRIPTION
**Pull request by mariaruth**
Small edits (repeated words).
Added subsection header for code examples.
Removed bullet points about good code.

### CODE REVIEW

File: chapters/handling-data.tex [hunk 1]
```diff
@@ -78,14 +86,8 @@ \subsection{Research reproducibility}
...
 	\url{https://blogs.worldbank.org/opendata/making-analytics-reusable}}
-Services that log your research process are valuable resources here --
-Such services can show things like modifications made in response to referee comments,
-by having tagged version histories at each major revision.
-They also allow you to use issue trackers
-to document the research paths and questions you may have tried to answer
-as a resource to others who have similar questions.
 
-Secondly, reproducible research\sidenote{
+Making your research reproducible is also a public good: \sidenote{
```
Comment (bbdaniels) [inline chapters/handling-data.tex]:
```suggestion
Making your research reproducible is also a public good.\sidenote{
```

File: chapters/handling-data.tex [hunk 2]
```diff
@@ -78,14 +86,8 @@ \subsection{Research reproducibility}
...
 	\url{https://dimewiki.worldbank.org/Reproducible_Research}}
 enables other researchers to re-use your code and processes
```
Comment (bbdaniels) [inline chapters/handling-data.tex]:
```suggestion
It enables other researchers to re-use your code and processes
```

File: chapters/handling-data.tex [hunk 3]
```diff
@@ -37,36 +37,44 @@
...
+	\url{https://www.aeaweb.org/journals/policies/data-code}}
+
 
 \subsection{Research reproducibility}
 
-Reproducible research means that the actual analytical processes you used are executable by others.\cite{dafoe2014science}
-(We use ``reproducible'' and ``replicable'' interchangeably in this book,
-though there is much discussion about the use and definition of these concepts.\sidenote{
+Can another researcher reuse the same code on the same data 
+and get the exact same results as in your published paper?\sidenote{	\url{https://blogs.worldbank.org/impactevaluations/what-development-economists-talk-about-when-they-talk-about-reproducibility}}
```
Comment (bbdaniels) [inline chapters/handling-data.tex]:
```suggestion
and get the exact same results as in your published paper?\sidenote{
  \url{https://blogs.worldbank.org/impactevaluations/what-development-economists-talk-about-when-they-talk-about-reproducibility}}
```

File: chapters/handling-data.tex [hunk 4]
```diff
@@ -228,7 +235,7 @@ \subsection{Research credibility}
 
 \section{Ensuring privacy and security in research data}
 
-Anytime you are collecting primary data in a development research project,
+Anytime you are working with raw data in a development research project,
```
Comment (bbdaniels) [inline chapters/handling-data.tex]:
```suggestion
Anytime you are working with original data in a development research project,
```

File: chapters/handling-data.tex [hunk 5]
```diff
@@ -37,36 +37,44 @@
...
-This is even more important in research using primary data,
-and therefore these are not byproducts but core components of research output.
+As a result, it is often the case that the data 
+researchers use for such studies has never been reviewed by anyone else,
+so it is hard for others to verify that it was 
+collected, handled, and analyzed appropriately.
+
+Reproducible and transparent methods are key to maintaining credibility 
+and avoiding serious errors. 
+This is particularly true for research that relies on new data sources,
```
Comment (bbdaniels) [inline chapters/handling-data.tex]:
```suggestion
This is particularly true for research that relies on original or novel data sources, 
```

File: chapters/handling-data.tex [hunk 6]
```diff
@@ -257,7 +265,7 @@ \section{Ensuring privacy and security in research data}
 with a set of governance standards known as ``The Common Rule''.\sidenote{
 	\url{https://www.hhs.gov/ohrp/regulations-and-policy/regulations/common-rule/index.html}}
 If you interact with European institutions or persons,
-you will also become familiar with ``GDPR'',\sidenote{
+you will also become familiar with the General Data Protection Regulation ``GDPR'',\sidenote{
```
Comment (bbdaniels) [inline chapters/handling-data.tex]:
```suggestion
you will also become familiar with the General Data Protection Regulation (GDPR),\sidenote{
```

File: chapters/handling-data.tex [hunk 7]
```diff
@@ -284,19 +292,19 @@ \subsection{Obtaining ethical approval and consent}
...
   \url{https://dimewiki.worldbank.org/IRB_Approval}}
-An IRB which has sole authority over your project is not always apparent,
+Which IRB has sole authority over your project is not always apparent,
 particularly if some institutions do not have their own.
 It is customary to obtain an approval from a university IRB
 where at least one PI is affiliated,
 and if work is being done in an international setting,
 approval is often also required
-from an appropriate institution subject to local law.
+from an appropriate local institution subject to the laws of the country.
```
Comment (bbdaniels) [inline chapters/handling-data.tex]:
```suggestion
from an appropriate local institution subject to the laws of the country where data originates.
```

File: chapters/handling-data.tex [hunk 8]
```diff
@@ -365,32 +365,30 @@ \subsection{Transmitting and storing data securely}
...
 It is often very simple to conduct planning and analytical work
-using a subset of the data that has anonymous identifying ID variables,
-and has had personal characteristics removed from the dataset altogether.
+using a subset of the data that has been \textbf{de-identified}.
 We encourage this approach, because it is easy.
 However, when PII is absolutely necessary to a task,
 such as implementing an intervention
 or submitting survey data,
-you must actively protect those materials in transmission and storage.
+you must actively protect that data in transmission and storage.
```
Comment (bbdaniels) [inline chapters/handling-data.tex]:
```suggestion
you must actively protect that information in transmission and storage.
```

File: chapters/handling-data.tex [hunk 9]
```diff
@@ -37,36 +37,44 @@
...
+Can another researcher reuse the same code on the same data 
+and get the exact same results as in your published paper?\sidenote{	\url{https://blogs.worldbank.org/impactevaluations/what-development-economists-talk-about-when-they-talk-about-reproducibility}}
+This is a standard known as \textbf{computational reproducibility}, 
+and it is an increasingly common requirement for publication.\sidenote{
 \url{https://www.nap.edu/resource/25303/R&R.pdf}})
-All your code files involving data cleaning, construction and analysis
+It is best practice to verify computational reproducibility before submitting a paper before publication. 
+This should be done by someone who is not on your research team, on a different computer, 
+using exactly the package of code and data files you plan to submit with your paper. 
+Code that is well-organized into a master do-file, and written to be easily run by others,
```
Comment (kbjarkefur) [inline chapters/handling-data.tex]:
```suggestion
Code that is well-organized into a master script, and written to be easily run by others, 
```

**Comment by kbjarkefur** [comment]
Apart from the discussion in https://github.com/worldbank/d4di/issues/385 I have no further comments on this PR

**Comment by mariaruth** [comment]
@bbdaniels and @kbjarkefur your comments/edits are all now addressed.

**Pull request merged**